migration_test.go

```
@@ -56,11 +58,12 @@ func TestMigration(t *testing.T) {
 
 	migrationsDir, migrationsDirErr := fs.Sub(testMigrationsDir, "testData")
 
-	assert.NoError(t, migrationsDirErr, "migrations directory could not be opened")
+	require.NoError(t, migrationsDirErr, "migrations directory could not be opened")
 
-	runErr := Run(db,
-		WithDialect(DialectSQLite()),
-		WithMigrationsFromFS(migrationsDir.(fs.ReadDirFS)))
+	runErr := dmorph.Run(db,
```

golangci-lint (2.2.2)

right hand must be only type assertion

(forcetypeassert)